app/controllers/account/admin/issues_controller.rb

```diff
@@ -7,7 +7,7 @@ class IssuesController < ApplicationController
       before_action :find_issue,
                     only: [:edit, :update, :approve, :close, :decline]
       def index
-        @issues = Issue.ordered.page(params[:page]).per(20)
+        issue_listing(Issue.all)
```

Comment:
the list of issues is formed by issue_listing method, it has sorting by 'created_at' field - https://github.com/Kotany-IF070-WebUI-RoR/NewVasjuky/pull/124/files#diff-127a850b7d7b20173ad3fba88f8a25e3